packages/@uppy/react/src/Dashboard.js

```diff
@@ -49,15 +51,16 @@ class Dashboard extends Component {
     uppy.removePlugin(this.plugin)
   }
 
+  [Symbol.for('htmlProps')] () { return this.#htmlProps }
```

Comment:
what's the purpose of this?

Reply:
Before:
https://github.com/transloadit/uppy/blob/7f5059b7843281d7d1adb9c33e5a06d232863beb/packages/%40uppy/react/src/nonHtmlPropsHaveChanged.js#L3-L10

After:
https://github.com/transloadit/uppy/blob/cf25ee5f9d16b19af0d3c42bc11bd316726cc6c3/packages/%40uppy/react/src/nonHtmlPropsHaveChanged.js#L3-L6

It's either this or refactoring the caller of the function to pass the private `#htmlProps` as an extra argument.

Reply:
I'm still not sure what's the benefit of having a `Symbol.for('htmlProps')` vs just a public property `htmlProps` on the class

Reply:
Yeah it doesn't make sense to use `Symbol.for` here, instead we want to define `export const kHtmlProps = Symbol('htmlProps')` on some internal file, and import it where we want to use it. `Symbol('htmlProps') !== Symbol('htmlProps')` (i.e. guaranteed to not clash with a user-defined symbol), but `Symbol.for('htmlProps') === Symbol.for('htmlProps')`.

Reply:
I simplified it. We used to pass the component to only reference the props, so instead it pass the props in the `componentDidUpdate` method and remove the need for an export.